README.md

```diff
@@ -1,8 +1,36 @@
 # BlaBlaCar React component library
 
-## CLI
-There is an internal set of commands available, to bkirk. To bootstrap our new componentst:
+## Installation
+BlaBlaCar `ui-library` is available as an [npm package](https://www.npmjs.com/package/@blablacar/ui-library)
 
+Just run this inside your project
 ```
-npm run kirk
+npm i @blablacar/ui-library
 ```
+
+## Usage
```

Comment:
You should also document how to use it for server side rendering with styled-jsx

Reply:
```
import { flushToHTML } from '@blablacar/ui-library'

const index = `
    <!doctype html>
    <html>
      <head>
        ${title}
        ${flushToHTML()}
      </head>
    ...
    `
```

Reply:
Agreed!

Reply:
Done